test/source/util/index.ts

```diff
@@ -71,6 +71,17 @@ export class Config {
     return Config.secrets().keys.filter(k => k.title === title)[0];
   }
 
+  public static setupSecrets = async (): Promise<void> => {
+    await Config.fixKeyInfo(Config._secrets);
+  }
+
+  public static fixKeyInfo = async (secrets: TestSecretsInterface): Promise<void> => {
```

Comment:
Please add a comment here. "The keys in test secrets file used to have different structure, this does a migration so that we can continue using the file as is without distributing an updated secrets file to everyone"

Reply:
Done